src/lib/y2network/widgets/route_options.rb

```diff
@@ -18,8 +18,10 @@ def label
       end
 
       def help
-        # TODO: original also does not have help
-        ""
+        _(
+          "<p><b>Options</b> specifies additional options for route. It is directly passed " \
+            "to <i>ip route add</i> with exception of <i>to</i>,<i>via</i> and <i>dev</i>."
```

Comment:
NP: i'd wirte something like "passed to routing" ... we don't need to expose particular command used - what if it changes in the future again ;-)

Reply:
well, that is useless from my POV. User need to know which options he can use. If we use generic route options, then we need to define what we support. If we specify command they know what they can use, if we do not specify it, they will find it themself and if we change it, it stop working for them, which is pretty annoying and not much friendly.